.github/ISSUE_TEMPLATE/3-slow-connection.yml

```diff
@@ -0,0 +1,118 @@
+name: Slow Connection to Flathub
+description: Report slow downloads from Flathub with diagnostic information
+title: "Slow downloads from Flathub (dl.flathub.org) — [Country/ISP]"
+labels: ["connectivity", "performance"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for reporting connectivity issues with Flathub. Please provide the diagnostic information below to help us investigate the problem.
+
+        For detailed instructions on collecting this information, see: [Slow connection to Flathub documentation](https://flathub.org/docs/for-users/slow-connection)
+
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: Describe what you observed (e.g., "Flatpak updates/downloads are very slow, ~50 KB/s, but other websites are fast")
+      placeholder: Describe the slow connection issue in one or two sentences
+    validations:
+      required: true
+
+  - type: input
+    id: datetime
+    attributes:
+      label: When it happened
+      description: Date and time when you experienced the issue (local time with timezone)
+      placeholder: "YYYY-MM-DD HH:MM TZ (e.g., 2025-11-05 14:30 CET)"
+    validations:
+      required: true
+
+  - type: input
+    id: os
+    attributes:
+      label: Operating System
+      description: Your distribution and version
+      placeholder: "e.g., Fedora 39, Ubuntu 24.04, etc."
+    validations:
+      required: true
+
+  - type: input
+    id: flatpak-version
+    attributes:
+      label: Flatpak Version
+      description: Output of `flatpak --version`
+      placeholder: "e.g., Flatpak 1.15.6"
+    validations:
+      required: true
+
+  - type: textarea
+    id: basic-connectivity
+    attributes:
+      label: Basic Connectivity Tests
+      description: |
+        Please run these commands and paste the output:
+        ```bash
+        ping -c 6 dl.flathub.org
+        dig +short dl.flathub.org
+        dig @8.8.8.8 dl.flathub.org
+        curl -sS -w "DNS: %{time_namelookup}s Connect: %{time_connect}s StartTransfer: %{time_starttransfer}s Total: %{time_total}s\n" -o /dev/null https://dl.flathub.org/repo/appstream/x86_64/appstream.xml.gz
+        traceroute dl.flathub.org
+        ```
+      placeholder: Paste the output from the connectivity tests
+      render: shell
+    validations:
+      required: true
+
+  - type: textarea
+    id: flatpak-output
+    attributes:
+      label: Flatpak Verbose Output
+      description: |
+        Attach the verbose output from a failing `flatpak update` or `flatpak install` run:
+        ```bash
+        flatpak update --verbose --ostree-verbose
+        ```
+
+        Or for a specific app:
+        ```bash
+        flatpak update --verbose --ostree-verbose org.gnome.Calculator
```

Comment:
this will fail if they don't have this installed, better to change it to an install command that installs some small app